### PR TITLE
Merge release/0.13.2 into develop branch

### PR DESCRIPTION
**Pull request by nikolaglumac**
This PR merges `release/0.13.2` code into the `develop` branch.

### CODE REVIEW

File: features/tests/e2e/setup/electron.js [hunk 1]
```diff
@@ -119,6 +119,19 @@ After({ tags: '@restartApp' }, async function() {
   context.app = await startApp();
 });
 
+// this ensures that the reset-backend call successfully executes
+// after the app version difference test sets the app to disconnected state
+// eslint-disable-next-line prefer-arrow-callback
+After({ tags: '@reconnectApp' }, async function() {
```
Comment (nikolaglumac) [inline features/tests/e2e/setup/electron.js]: @daniloprates @tomothespian @MarcusHurney @DominikGuzei I have added this new tag to our e2e setup as new test for "Manual software update overlay" pushes Daedalus into disconnected state which causes all the tests which are ran after it to fail due to a fact that `Backend-Reset` Before step call fails (it expects Daedalus to be connected).

Comment (daniloprates) [reply features/tests/e2e/setup/electron.js]: ✅

**Comment by nikolaglumac** [comment]
@daniloprates @tomothespian @MarcusHurney @DominikGuzei there were some issues with the app-version check logic which I have discovered tonight and decided to fix them in the scope of this PR: https://github.com/input-output-hk/daedalus/pull/1421/commits/1241015d19e192711aa7c3e264e7a8310a76a1e1
I will port most of these improvements to the `release/0.13.2` branch in a separate PR.
The issues which were fixed by these changes are:
- Proper handling of a case where app-version API call fails (in which case we assume there is no newer version available and show the "Having trouble connecting/syncing" notification)
- Proper handling of Windows platform version check (original call was trying to fetch info for `platforms.win32.version` and the API responds with `platforms.windows.version`) - this was a major issue as on Windows we would never get version info 😱 
- Simplification of the `isNewAppVersionLoading` variable by reusing Request.wasExecuted and Request.isExecuting flags

**Comment by nikolaglumac** [comment]
@DeeJayElly here you can see how I have made sure the ManualUpdate overlay is hidden during Node shutdown sequence. I thought this might be useful for you :)

**Comment by nikolaglumac** [comment]
@daniloprates @tomothespian @MarcusHurney I have decided to port all the fixes onto the `release/0.13.2` branch: https://github.com/input-output-hk/daedalus/pull/1423
Please review and approve that PR first. After we get that one merged I will update this PR again...

**Comment by nikolaglumac** [comment]
Release `0.13.2` has been canceled and the version `0.13.2` removed from the `CHANGELOG` and `package.json` in: https://github.com/input-output-hk/daedalus/pull/1441

**Pull request merged**